mobilecoind/strategies/drain-accounts.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+
+"""
+The purpose of this strategy is to drain funds from one account to another.
+This is one way to transfer tokens to a fog account, which were created after bootstrap.
+This is the case for minted tokens (which fog distro cannot transfer).
+
+This script thus provides a method for testing transactions with minted tokens, with fog wallets.
+
+It zips the source accounts and dest accounts together, and each source account sends
+its entire balance (less fee) to the dest account
+
+(Currently the process of using mc-util-generate-sample-ledger to generate an origin block
+and then sending transactions via mc-fog-distribution is blind to minted tokens because
+mc-fog-distribution only sends transactions based on initial balances generated in the sample
+ledger.)
+
+Example setup and usage:
+```
+    python3 drain-accounts.py --key-dir ../../../target/sample_data/master/keys/ --dest-key-dir ../../../target/sample_data/master/fog_keys/ --fee 20
+```
+"""
+import argparse
+import grpc
+import mobilecoind_api_pb2
+import mobilecoind_api_pb2_grpc
+import os
+import time
+from accounts import connect, load_key_and_register, poll, wait_for_accounts_sync, TransferStatus
+from google.protobuf.empty_pb2 import Empty
+
+
+def parse_args() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--mobilecoind-host",
+                        default="localhost",
+                        type=str,
+                        help="Mobilecoind host")
+    parser.add_argument("--mobilecoind-port",
+                        default="4444",
+                        type=str,
+                        help="Mobilecoind port")
+    parser.add_argument("--key-dir",
+                        type=str,
+                        help="Path to directory of account_keys")
+    parser.add_argument("--dest-key-dir",
+                        type=str,
+                        help="Path to directory of destination account_keys")
+    parser.add_argument("--max-seconds",
+                        type=int,
+                        default=40,
+                        help="Number of seconds to wait for a tx to clearn")
+    parser.add_argument("--fee",
+                        type=int,
+                        default=20,
+                        help="Amount less than the balance that we attempt to send")
```

Comment:
This sounds incorrect to me, isn't this the fee and not the amount we are trying to send?

Reply:
yeah i just copied these comments from the other script

i think we attempt to send an amount which is 'balance - fee' with a fee value of fee, so the comment is not wrong, it's just wierd.

side note: idk if there is a way to ask mobilecoind what the fee is? then we would not need to have this parameter